plugins/todo-backend/api-report.md

```diff
@@ -131,6 +131,7 @@ export type TodoScmReaderOptions = {
   reader: UrlReader;
   integrations: ScmIntegrations;
   parser?: TodoParser;
+  excludeFolders?: string[];
```

Comment:
What you think about making this an optional `filePathFilter` instead? A `(filePath: string) => boolean` callback that we add to the filtering logic instead. That was you're not locked into only providing prefixes and get a bit more flexibility.

I think with that we could also start by not having this configured through `app-config.yaml`, I don't really see any immediate use-case for having this change based on different deployment environment. Better make it more flexible and keep the API surface low for now.